### PR TITLE
Added Chart Of Accounts for Coco Products

### PR DESCRIPTION
**Pull request by suhasagubbi**
Added Chart Of Accounts for Coco Products

### CODE REVIEW

**Review by suhasagubbi — COMMENTED** [review]
OK

File: Chart_of_Accounts.csv [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+10053,"3010",SGST @ 0%,A,L,AR_tax:AP_tax:IC_taxpart:IC_taxservice,"",false,false
+10142,"3060",CGST @ 6%,A,L,AR_tax:AP_tax:IC_taxpart:IC_taxservice,"",false,false
+10147,"3100",CGST @ 14%,A,L,AR_tax:AP_tax:IC_taxpart:IC_taxservice,"",false,false
+10003,"1065",Cash In Hand,A,A,AR_paid:AP_paid,"",false,false
+10119,"2110",Bank Account - Syndicate Bank OD,A,L,AR_paid:AP_paid,"",false,false
+10050,"2120",Hand Loan - Krishna H S,A,L,"","",false,false
+10156,"2130",Hand Loan - Sudhira H S,A,L,"","",false,false
+10157,"2140",Hand Loan - Suhasa H S,A,L,"","",false,false
+10158,"2150",Hand Loan - Bharata H S,A,L,"","",false,false
+10017,"2630",Partners H S K & Co,A,L,"","",false,false
```
Comment (suhasagubbi) [inline Chart_of_Accounts.csv]:
```suggestion
10017,"2630",H S K & Co,A,L,"","",false,false
```

**Review by suhasagubbi — COMMENTED** [review]
pl,pl

**Review by bsrinivasa — APPROVED** [review]
👍

**Pull request merged**